streams/src/main/java/org/apache/kafka/streams/state/internals/InMemoryKeyValueStoreSupplier.java

```diff
@@ -88,17 +85,9 @@ public String name() {
         }
 
         @Override
+        @SuppressWarnings("unchecked")
         public void init(ProcessorContext context, StateStore root) {
-            if (loggingEnabled) {
-                context.register(root, true, new StateRestoreCallback() {
-
-                    @Override
-                    public void restore(byte[] key, byte[] value) {
-                        put(serdes.keyFrom(key), serdes.valueFrom(value));
-                    }
-                });
-
-            }
+            // do nothing
```

Comment:
Nitpick: In most cases "do nothing" comments are kinda unnecessary because you can clearly see that the method body is empty, thus "doing nothing".  I'd rather explain *why* it isn't doing anything.

Idea:

```java
// InMemoryKeyValueStoreSupplier does not need any initialization.
```